Use ENV variables for service names in the config

bdunne opened the pull request:
Depends on https://github.com/ManageIQ/manageiq-pods/pull/176

carbonin commented on docker-assets/manageiq.conf:
```diff
@@ -67,8 +67,8 @@ RewriteRule ^.*$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R]
   RewriteRule ^(.*)$ $1/ [R,NE,L]
-  ProxyPass /ansibleapi/ http://localhost:54321/api/
-  ProxyPassReverse /ansibleapi/ http://localhost:54321/api/
+  ProxyPass /ansibleapi/ http://#{ANSIBLE_SERVICE_NAME}/api/
```
I assume this should be `${ANSIBLE_SERVICE_NAME}`

carbonin replied:
Also, I don't think we route from the apache pod to ansible in containers (at least not on purpose). We hit the ansible service directly. Are we expecting to expose that service to people who are not our servers?

bdunne replied:
Fixed.  I thought it was a requirement to expose the Ansible API externally.

@Fryguy or @gtanzillo Do you know if we need to expose this to the outside world as we did in VM's?

Fryguy replied:
We only had to expose to the outside world because the other appliances needed to talk to it via the external network.  Inside of a kube cluster I would expect they would be in the same project and thus it can just hit the service directly.

carbonin replied:
Okay, so I think this can all go away then.

carbonin reviewed (changes requested):
I didn't know apache configs would interpolate environment variables. I figured we would have to go back to dynamic config files.

This is great!

bdunne committed:
Use ENV variables for service names in the config

Fryguy commented:
😍

bdunne committed:
Drop all Ansible API references

These were added for embedded ansible, but since we're in a project, all
traffic will go directly to the service, there should be no inbound
requests for /ansibleapi
